upgrade to checkstyle 9.0

https://checkstyle.org/releasenotes.html#Release_9.0

I am wondering if there is any update of the sonar-checkstyle plugin. 

I am interested in this issue https://github.com/checkstyle/checkstyle/issues/8050 that seems it has been solved in version 9.0

I am trying to scan a code that uses methods and variables with Spanish character 'ñ' or 'Ñ' since I get the following exception:

Caused by: com.puppycrawl.tools.checkstyle.api.CheckstyleException: TokenStreamRecognitionException occurred while parsing file

Or is there way to add any kind of filter to filter out this kind of minor issues?

--- a/README.md
+++ b/README.md
@@ -21,6 +21,7 @@ Compatibility matrix from checkstyle team:
 
 Checkstyle Plugin|Sonar min|Sonar max|Checkstyle|Jdk
 -----------------|---------|---------|----------|---
+9.0.1|8.9  |8.9+|9.0.1|1.8
 8.45.1|7.9  |7.9+|8.45.1|1.8
 8.42|7.9  |7.9+|8.42|1.8
 8.41.1|7.9  |7.9+|8.41.1|1.8

--- a/pom.xml
+++ b/pom.xml
@@ -93,7 +93,7 @@
   </ciManagement>
 
   <properties>
-    <checkstyle.version>8.45.1</checkstyle.version>
+    <checkstyle.version>9.0.1</checkstyle.version>
     <sonar.version>8.9.3.48735</sonar.version>
     <sonar-java.version>7.2.0.26923</sonar-java.version>
     <maven.checkstyle.plugin.version>3.1.2</maven.checkstyle.plugin.version>

